src/tests/unittests/validation/RenderCommandsValidationTests.cpp

```diff
@@ -0,0 +1,258 @@
+// Copyright 2017 The NXT Authors
```

Comment:
Can this be in a VertexBufferValidationTests.cpp file and have more stuff factored (shader creation, creation of a bunch of vertex buffers, etc) so we can easily update it if the API changes, and easily add more tests?